### PR TITLE
change migration

### PR DESCRIPTION
**Pull request by VLiamin**

### CODE REVIEW

**Review by MariyaSin — APPROVED** [review]

**Review by Cheephot — APPROVED** [review]

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+               },
+               constraints: table =>
+               {
+                   table.PrimaryKey("PK_EntitiesImages", x => x.Id);
+               });
+        }
+
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropTable(DbEntityImage.TableName);
```
Comment (BelehovEgor) [inline src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs:30]: is it initial migration? why do you want to drop db?

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    [Migration("20210824232700_AddEntitiesImagesTable")]
+    public class AddEntitiesImagesTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+               name: DbEntityImage.TableName,
+               columns: table => new
+               {
+                   Id = table.Column<Guid>(nullable: false),
```
Comment (BelehovEgor) [inline src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs:18]: fix tabs

Comment (BelehovEgor) [reply src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs:18]: line 15 - 3, here 4

**Review by BelehovEgor — APPROVED** [review]

File: src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs (hunk 2 above)
Comment (VLiamin) [reply src/ProjectService.Data.Provider.MsSql.Ef/Migrations/20210824232700_AddEntitiesImagesTable.cs]: done

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**